java/arcs/core/storage/ReferenceModeStore.kt

```diff
@@ -96,6 +97,7 @@ class ReferenceModeStore private constructor(
     /* internal */
     val backingKey: StorageKey,
     /* internal */
+    clearCoroutineContext: CoroutineContext = Dispatchers.Default,
```

Comment:
Move this into options?

Reply:
I want to make it clear this one is used for clearing things in ReferenceModeStore.kt. There are other coroutine context used in ReferenceModeStore.kt.

Reply:
Fair enough